applications/quantumleap/chart/values.yaml

```diff
@@ -14,17 +14,17 @@ quantumleap:
       tag: 0.8.3
   image:
     # -- Container image to deploy
-    repository: docker.io/orchestracities/quantumleap
+    repository: quay.io/fiware/quantumleap
```

Comment:
No, the image is build by orchestractities in this ci: https://github.com/orchestracities/ngsi-timeseries-api/actions/runs/2292760549
But since its only available with the "edge"-tag, I decided to retag it, so that we do not get it silently replaced by the next ci-run.